lib/ts/recipe/emailverification/api/implementation.ts

```diff
@@ -17,7 +18,17 @@ export default function getAPIInterface(): APIInterface {
             const res = await options.recipeImplementation.verifyEmailUsingToken({ token, userContext });
 
             if (res.status === "OK" && session !== undefined) {
-                await session.fetchAndSetClaim(EmailVerificationClaim, userContext);
+                try {
+                    await session.fetchAndSetClaim(EmailVerificationClaim, userContext);
+                } catch (err) {
+                    // This should never happen, since we've just set the status above.
```

Comment:
what if a session does not exist? Then what affect will throwing this error have?

Reply:
Then we can't have a `UNKOWN_USER_ID` thrown since that's only from `fetchAndSetClaim`.
In the case of a deleted user, it'll return `EMAIL_VERIFICATION_INVALID_TOKEN_ERROR` if the token has already been revoked, otherwise, it'll succeed.